core/modules/dashboard/includes/block.overview_content.inc

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @file
+ *
+ * Dashboard block which displays a summary of content statistics including:
+ * - number pieces of each type of content
+ * - number of comments
+ * - number of blocked comments (spam)
+ * - configuration links
+ */
+class DashboardOverviewContentBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Content');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Overview of content');
+  }
+
+  /**
+   * 'Admin info' callback for the content type.
+   */
+  function getAdminPreview() {
+    return t('Displays a summary of content statistics.');
+
+  }
+
+  /**
+   * Run-time rendering of the body of the block.
+   */
+  function getContent() {
+    $items = dashboard_get_content_overview($this->settings);
+    return theme('dashboard_overview_content', array('overview' => $items));
+
+  }
+
+  /**
+   * 'Edit form' callback for the content type.
+   */
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+    // Add content settings to form.
+    dashboard_add_content_block_settings($form, $this->settings);
```

Comment:
Looks like this is only called in this one place. Couldn't it just be embedded directly rather than calling another function?

Reply:
fixed by @quicksketch